study.md

```diff
@@ -29,21 +29,37 @@ Asynchronous - operations that may start in a certain sequence and may end in an
 In your own words, describe the event loop and how it works.
 
 ```md
-<!-- your answer here -->
+<!-- Using readings/videos required for this homework and trying to remember diffs b/w
+heaps/stacks (man, that was a long time ago), I understand the event loop
+to be a single-dimensional data structure that handles commands one at
+a time.  Each line of code is loaded onto a stack and popped up as it is
+handled... LIFO and this ends up taking some time because the prior command/task
+needs to finish before the next one begins (--and who has time for that?!) -->
```

Comment:
Yes so the event loop handles the async code. If the call stack is empty, the event loop will take the next thing from the queue and move it to the stack.